api/envoy/extensions/transport_sockets/tls/v3/common.proto

```diff
@@ -253,7 +254,21 @@ message CertificateProviderPluginInstance {
   string certificate_name = 2;
 }
 
-// [#next-free-field: 15]
+// String matcher for subject alternative names, to match both type and value of the SAN.
+message SubjectAltNameMatcher {
+  enum SanType {
```

Comment:
Please add a comment here with the link to the relevant RFC section.

Reply:
Added comment.